king_phisher/client/application.py

```diff
@@ -552,6 +555,17 @@ def do_config_load(self, load_defaults):
 			for key, value in client_template.items():
 				if not key in self.config:
 					self.config[key] = value
+			env_proxy = os.environ.get('HTTPS_PROXY')
+			if env_proxy is not None:
+				proxy_url = urllib.parse.urlparse(env_proxy)
+				if proxy_url.hostname and not proxy_url.scheme or proxy_url.scheme and not proxy_url.hostname:
```

Comment:
You could simplify this to no `if not (proxy_url.hostname and proxy_url.scheme)` and I think it'd be a bit more readable too.